lib/notification/notificationcomponent.ti

```diff
@@ -9,7 +9,7 @@ namespace icinga
 
 class NotificationComponent : ConfigObject
 {
-	activation_priority 200;
+	activation_priority 75;
```

Comment:
https://github.com/Icinga/icinga2/pull/10191#discussion_r1836911127 sounds like you'd like to do the opposite, i.e to leave this as-is. Imagine, all data outputs (100) are shut down or not yet online, but notifications (75) get sent – silently.

Reply:
https://github.com/Icinga/icinga2/pull/10191#discussion_r1837781557 This was your suggestion.... I honestly find this code so confusing that I just accepted it as is.

Reply:
Sorry, I was fully focused on cleaning up the diff.
Anyway, now after thinking about what's left, _I suspect_ what matters in your case is Notification -5 and this one is not needed or even harmful.